### PR TITLE
update numberjs and tests

### PR DESCRIPTION
**Pull request by dkraytsberg**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -24,7 +24,8 @@ module.exports = {
     'react/require-default-props': 0,
     // https://github.com/yannickcr/eslint-plugin-react/issues/885
     'react/no-unused-prop-types': 0, // there are bugs with this and stateless functions
-    'no-mixed-operators': 0
+    'no-mixed-operators': 0,
+    'no-bitwise': 0,
```
Comment (ntenczar) [inline .eslintrc.js]: honestly we shouldn't even use that pseudoUUID crap. Just remove it for now and get rid of this rule

Comment (ntenczar) [inline .eslintrc.js]: I believe it only gets used for places? not a concern for now

Comment (dkraytsberg) [inline .eslintrc.js]: test-push-integration-state.js

Comment (ntenczar) [inline .eslintrc.js]: yeah that's only for old angular rubbish don't worry about it

**Pull request merged**